apps/testingaccessibility/src/pages/thanks/purchase.tsx

```diff
@@ -44,27 +44,7 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
 
   const {email, stripeChargeId, quantity: seatsPurchased} = purchaseInfo
 
-  // Find the purchase associated with this stripeChargeId
-  // - Is the tied to a bulk coupon?
-  //   - If so, is it the first one?
-  //     - Yes: NEW_BULK_COUPON
-  //     - No:  EXISTING_BULK_COUPON
-  //   - If not: NEW_INDIVIDUAL_PURCHASE
-
-  const purchase = await prisma.purchase.findFirst({
-    where: {
-      merchantCharge: {
-        identifier: stripeChargeId,
-      },
-    },
-    include: {
-      bulkCoupon: {
-        include: {
-          bulkCouponPurchases: true,
-        },
-      },
-    },
-  })
+  const purchase = await getSdk().getPurchaseForStripeCharge(stripeChargeId)
```

Comment:
I'd prefer to use consistent destructuring syntax everywhere instead of this chaining approach if possible

`const {getPurchaseForStripeCharge} = getSdk()`

even when there is "only one method being used" it introduces an alternative structure which is confusing to parse for me